include/swift/AST/Decl.h

```diff
@@ -2353,6 +2361,8 @@ class ValueDecl : public Decl {
     LazySemanticInfo.hasOverridden = false;
     LazySemanticInfo.isDynamicComputed = false;
     LazySemanticInfo.isDynamic = false;
+    LazySemanticInfo.isFinalComputed = false;
+    LazySemanticInfo.isFinal = false;
```

Comment:
I wonder if we should add a default initializer to LazySemanticInfo instead, to avoid mistakes where we forget to add something here.

Reply:
I actually made this mistake on my first try. I've added a commit replacing this section with a default initializer, please let me know if it is what you had in mind.